src/components/ShoppingCart/ShoppingCartContainer.tsx

```diff
@@ -16,6 +16,7 @@ import { ICartItem } from "@/interfaces/cartItem";
 import DeleteComfirmModal from "@/components/DeleteComfirmModal";
 import { useDeleteItemFromCartMutation } from "@/redux/api/cartApi";
 import { RiErrorWarningLine } from "react-icons/ri";
+import { deleteImage } from "@/utils/uploadImage";
```

Comment:
If you have deleteImage method in this util, you should not name it as uploadImage.

Reply:
It's named manageExternalImage now